orchestration/dags/dependencies/firebase/sql/analytics/firebase_home_events.sql

```diff
@@ -0,0 +1,167 @@
+WITH mapping_module_name_and_id AS (
+    SELECT
+        *
+    FROM
+        (
+            SELECT
+                relations.parent as module_id,
+                entries.title as module_name,
+                row_number() over (
+                    partition by entries.title
+                    order by
+                        updated_at DESC
+                ) as rnk
+            from
+                `{{ bigquery_analytics_dataset }}.contentful_relationships` relations
+                inner join `{{ bigquery_analytics_dataset }}.contentful_entries` entries on entries.id = relations.child
+            WHERE
+                entries.content_type in (
+                    "displayParameters",
+                    "venuesSearchParameters",
+                    "exclusivityDisplayParameters",
+                    "business"
+                )
+                
+        )
+    where
+        rnk = 1
+),
+firebase_events AS (
+    SELECT
+        coalesce(e.module_id, c_name.module_id) as module_id,
+        e.*
+    EXCEPT
+(module_id, module_name)
+    FROM
+        `{{ bigquery_analytics_dataset }}.firebase_events` e
+        LEFT JOIN mapping_module_name_and_id c_name on c_name.module_name = e.module_name
+    WHERE
+        event_name in (
+            'ConsultOffer',
+            'BusinessBlockClicked',
+            'ExclusivityBlockClicked',
+            'ModuleDisplayedOnHomePage'
+        )
+        AND firebase_screen = "Home"
```

Comment:
J'ai l'impression qu'on ne peut pas trop se baser sur le firebase_screen, exemple pour l'event `ModuleDisplayedOnHomePage ` on a plusieurs valeurs possibles de firebase_screen alors que logiquement le seul screen logique devrait être 'Home'

On pourrait mettre une condition type `AND (origin = 'home' OR origin IS NULL)`, puisque : 
- cela semble plus robuste pour l'event ConsultOffer (car paramétré par nos devs)
- les events BusinessBlockClicked, ExclusivityBlockClicked et ModuleDisplayedOnHomePage viennent forcément de la Home